docs/sources/configure-client/grafana-agent/ebpf/troubleshooting.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Troubleshooting eBPF installation"
+menuTitle: "Troubleshooting"
+description: "Troubleshooting Grafana eBPF installation"
+weight: 40
+---
+
+# Troubleshooting eBPF installation
+
+## Profiling interpreted languages
+
+Profiling interpreted languages like Ruby, JavaScript, etc., is not ideal using this implementation.
+The JIT-compiled methods in these languages are typically not in ELF file format, demanding additional steps for
+profiling. For instance, using perf-map-agent and enabling frame pointers for Java.
+
+Interpreted methods will display the interpreter function’s name rather than the actual function.
+
+## Troubleshooting unknown symbols
+
+Symbols are extracted from various sources, including:
+
+* The `.symtab` and `.dynsym` sections in the ELF file.
+* The `.symtab` and `.dynsym` sections in the debug ELF file.
+* The `.gopclntab` section in Go language ELF files.
+
+The search for debug files follows [gdb algorithm](https://sourceware.org/gdb/onlinedocs/gdb/Separate-Debug-Files.html).
+For example, if the profiler wants to find the debug file
+for `/lib/x86_64-linux-gnu/libc.so.6`
+with a `.gnu_debuglink` set to `libc.so.6.debug` and a build ID `0123456789abcdef`. The following paths are examined:
+
+* `/usr/lib/debug/.build-id/01/0123456789abcdef.debug`
+* `/lib/x86_64-linux-gnu/libc.so.6.debug`
+* `/lib/x86_64-linux-gnu/.debug/libc.so.6.debug`
+* `/usr/lib/debug/lib/x86_64-linux-gnu/libc.so.6.debug`
+
+### Dealing with unknown symbols
+
+Unknown symbols in the profiles you’ve collected indicate that the profiler couldn't access an ELF file associated with a given address in the trace.
+
+This can occur for several reasons:
+
+* The process has terminated, making the ELF file inaccessible.
+* The ELF file is either corrupted or not recognized as an ELF file.
+* There is no corresponding ELF file entry in `/proc/pid/maps` for the address in the stack trace.
+
+### Addressing unresolved symbols
+
```

Comment:
```suggestion
### Deal with unknown symbols

Unknown symbols in the profiles you’ve collected indicate that the profiler couldn't access an ELF file associated with a given address in the trace.

This can occur for several reasons:

* The process has terminated, making the ELF file inaccessible.
* The ELF file is either corrupted or not recognized as an ELF file.
* There is no corresponding ELF file entry in `/proc/pid/maps` for the address in the stack trace.

### Address unresolved symbols

```